orb-slot-ctrl/src/main.rs

```diff
@@ -82,120 +95,68 @@ fn main() -> eyre::Result<()> {
             println!("{}", orb_slot_ctrl::get_next_boot_slot()?);
         }
         Commands::SetNextSlot { slot } => {
-            let slot = match slot.as_str() {
-                // Slot A alias.
-                "A" => orb_slot_ctrl::Slot::A,
-                "a" => orb_slot_ctrl::Slot::A,
-                "0" => orb_slot_ctrl::Slot::A,
-                // Slot B alias.
-                "B" => orb_slot_ctrl::Slot::B,
-                "b" => orb_slot_ctrl::Slot::B,
-                "1" => orb_slot_ctrl::Slot::B,
-                _ => {
-                    println!(
-                        "Invalid slot provided, please use either A/a/0 or B/b/1."
-                    );
-                    exit(1)
-                }
-            };
             if let Err(e) = orb_slot_ctrl::set_next_boot_slot(slot) {
                 check_running_as_root(e);
             };
         }
-        Commands::Status { inactive, subcmd } => {
-            match subcmd {
-                StatusCommands::GetRootfsStatus => {
-                    if inactive {
-                        println!(
-                            "{:?}",
-                            orb_slot_ctrl::get_rootfs_status(
-                                orb_slot_ctrl::get_inactive_slot()?
-                            )?
-                        );
-                    } else {
-                        println!("{:?}", orb_slot_ctrl::get_current_rootfs_status()?);
-                    }
+        Commands::Status { inactive, subcmd } => match subcmd {
```

Comment:
cargo fmt formatted this for me. are there any other configs apart from https://github.com/worldcoin/orb-software/blob/main/rustfmt.toml ?

Reply:
Nope, that file is the canonical source of truth. Might be that a rust update improved the formatter? Or do you have some sort of formatting override set up by accident? CI will be the final arbiter here.

Reply:
Let me know if there are more changes needed or otherwise i will squash commits

Reply:
No further changes needed.